components/oneapi/src/main/java/com/wso2telco/dep/oneapi/constant/payment/PaymentKeyConstants.java

```diff
@@ -0,0 +1,14 @@
+package com.wso2telco.dep.oneapi.constant.payment;
+
+/**
+ * This class hold the payment api related payload  keys as constants.
+ *
+ * @author Salinda Karunarathna
```

Comment:
Usually with Apache License we do not encourage using Author tags :)

Reply:
Are there any finalized license header to use?